### ISSUE TITLE
Keyboard shortcuts - move zoomed picture

### ISSUE THREAD

**Issue by xerostomus** [onlyshk/GPicView#6]
As I know there is no keyboard shortcut for moving a zoomed picture. 
So I suggest:

CTRL-arrows moves zoomed picture hither and thither...

Thank you

JK

**Comment by Jirido** [comment]
I agree. Also it would be logical if the zoom would zoom from the middle instead of from the upper left corner. Thanks.
